### PR TITLE
Add description of can't mix multiple backend

### PR DESCRIPTION
**Pull request by sunzhaoyang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-backends.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ TiDB Lightning 的[后端](/tidb-lightning/tidb-lightning-glossary.md#backend)
 | 支持 TiDB 集群版本 | >= v4.0.0 | 全部 | 全部 |
 | 是否影响 TiDB 对外提供服务 | 是 | 是 | 否 |
 
+> **注意：**
+>
+> 严禁使用多个 Lightning 进程向同一目标导入时混用不同的 backend，例如同时使用 local-backend 和 tidb-backend 导入同一 TiDB 集群是不允许的。
```
Comment (glorv) [inline tidb-lightning/tidb-lightning-backends.md:36]: 我感觉这里应该说明一下，在默认情况下，不应该启动多个 TiDB Lightning 向同一个下游集群导入数据，如果需要的话，应该参考 并行导入 章节的内容。

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 2]
```diff
@@ -16,7 +16,11 @@ TiDB Lightning 并行导入可以用于以下场景：
 
 >**注意：**
 >
-> 并行导入只支持初始化 TiDB 的空表，不支持导入数据到已有业务写入的数据表，否则可能会导致数据不一致的情况。
+> 1. 并行导入只支持初始化 TiDB 的空表，不支持导入数据到已有业务写入的数据表，否则可能会导致数据不一致的情况。
+> 
+> 2. 并行导入一般用于 local-backend 模式，虽然在 tidb-backend 模式下也可以工作，但并无太大意义。
```
Comment (glorv) [inline tidb-lightning/tidb-lightning-distributed-import.md:21]: tidb-backend 在某些情况下可能是有意义的，如果确认下游 tidb 不是瓶颈的话，我倾向于把 tidb backend 和 local backend 对于并行导入的支持区分开。不过考虑到 tidb backend 需要并行的场景应该有限，我们可以在一个小的章节里面简单提一下就好。tidb backend 的 并行依然需要满足部署限制里面的要求。

File: tidb-lightning/tidb-lightning-backends.md (hunk 1 above)
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-backends.md:36]:
```suggestion
> 默认情况下，不使用多个 TiDB Lightning 向同一个下游集群导入数据，如果特殊情况需要，可参考[并行导入](/tidb-lightning/tidb-lightning-distributed-import.md)。
> 使用多个 TiDB Lightning 向同一目标导入时，禁止混用不同的 backend，例如，不可同时使用 Local-backend 和 TiDB-backend 导入同一 TiDB 集群。
```

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 ---
-title: TiDB Lightning 分布式并行导入
-summary: 本文档介绍了 TiDB Lightning 分布式并行导入的概念、使用场景、使用说明和使用方法。
+title: TiDB Lightning 并行导入
+summary: 本文档介绍了 TiDB Lightning 并行导入的概念、使用场景、使用说明和使用方法。
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-distributed-import.md:3]:
```suggestion
summary: 本文档介绍了 TiDB Lightning 并行导入的概念、使用场景和使用方法。
```

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 4]
```diff
@@ -16,7 +16,11 @@ TiDB Lightning 并行导入可以用于以下场景：
 
 >**注意：**
 >
-> 并行导入只支持初始化 TiDB 的空表，不支持导入数据到已有业务写入的数据表，否则可能会导致数据不一致的情况。
+> 1. 并行导入只支持初始化 TiDB 的空表，不支持导入数据到已有业务写入的数据表，否则可能会导致数据不一致的情况。
+> 
+> 2. 并行导入一般用于 local-backend 模式，虽然在 tidb-backend 模式下也可以工作，但并无太大意义。
+>
+> 3. 严禁使用多个 Lightning 进程向同一目标导入时混用不同的 backend，例如同时使用 local-backend 和 tidb-backend 导入同一 TiDB 集群是不允许的。
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-distributed-import.md:23]:
```suggestion
> 3. 使用多个 TiDB Lightning 向同一目标导入时，禁止混用不同的 backend，例如，不可同时使用 Local-backend 和 TiDB-backend 导入同一 TiDB 集群。
```

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 5]
```diff
@@ -16,7 +16,11 @@ TiDB Lightning 并行导入可以用于以下场景：
 
 >**注意：**
 >
-> 并行导入只支持初始化 TiDB 的空表，不支持导入数据到已有业务写入的数据表，否则可能会导致数据不一致的情况。
+> 1. 并行导入只支持初始化 TiDB 的空表，不支持导入数据到已有业务写入的数据表，否则可能会导致数据不一致的情况。
+> 
+> 2. 并行导入一般用于 local-backend 模式。
+>
+> 3. 使用多个 TiDB Lightning 向同一目标导入时，禁止混用不同的 backend，例如，不可同时使用 Local-backend 和 TiDB-backend 导入同一 TiDB 集群.
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-distributed-import.md:23]:
```suggestion
> - 并行导入只支持初始化 TiDB 的空表，不支持导入数据到已有业务写入的数据表，否则可能会导致数据不一致的情况。
> 
> - 并行导入一般用于 local-backend 模式。
>
> - 使用多个 TiDB Lightning 向同一目标导入时，禁止混用不同的 backend，例如，不可同时使用 Local-backend 和 TiDB-backend 导入同一 TiDB 集群。
```

File: tidb-lightning/tidb-lightning-backends.md [hunk 6]
```diff
@@ -31,6 +31,11 @@ TiDB Lightning 的[后端](/tidb-lightning/tidb-lightning-glossary.md#backend)
 | 支持 TiDB 集群版本 | >= v4.0.0 | 全部 | 全部 |
 | 是否影响 TiDB 对外提供服务 | 是 | 是 | 否 |
 
+> **注意：**
+>
+> - 使用多个 TiDB Lightning 向同一目标导入时，禁止混用不同的 backend，例如，不可同时使用 Local-backend 和 TiDB-backend 导入同一 TiDB 集群.
+> - 默认情况下不应同时启动多个 Lightning 实例导入同一 TiDB 集群，应考虑使用并行导入特性。
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-backends.md:37]:
```suggestion
> - 使用多个 TiDB Lightning 向同一目标导入时，禁止混用不同的 backend，例如，不可同时使用 Local-backend 和 TiDB-backend 导入同一 TiDB 集群。
>
> - 默认情况下，不应同时启动多个 TiDB Lightning 实例导入同一 TiDB 集群，而应考虑使用并行导入特性。
```

**Review by shichun-0415 — APPROVED** [review]

**Review by glorv — APPROVED** [review]

**Pull request merged**